airbyte-integrations/base-java/src/main/java/io/airbyte/integrations/javabase/Command.java

```diff
@@ -22,8 +22,12 @@
  * SOFTWARE.
  */
 
-package io.airbyte.commons.functional;
```

Comment:
this isn't actually a rename. `CloseableConsumer` was unused so I removed it. the `Command` enum is new.